spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingAdapterRunner.java

```diff
@@ -17,16 +17,15 @@
 package org.springframework.cloud.stream.config;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
 import org.springframework.cloud.stream.adapter.ChannelBindingAdapter;
-import org.springframework.context.annotation.Configuration;
 
 /**
  * @author Dave Syer
  *
  */
-@Configuration
-public class LifecycleConfiguration implements CommandLineRunner {
+public class ChannelBindingAdapterRunner implements ApplicationRunner {
```

Comment:
My reasoning for not removing `@Configuration` (even `@Component`) in this case is that this specific class isn't a configuration bean and this isn't component scanned as it is imported in EnableModule annotation. I was even hoping to change EnableModule to use `@ImportResource` for non-configuration beans but kept it intact.